packages/@aws-cdk/aws-dynamodb/lib/table.ts

```diff
@@ -442,6 +463,16 @@ export class Table extends Construct {
     this.grant(principal, ...READ_DATA_ACTIONS);
   }
 
+  /**
+   * Permis an IAM principal all stream data read operations for this
+   * table's stream:
+   * DescribeStream, GetRecords, GetShardIterator, ListStreams.
+   * @param principal The principal to grant access to
+   */
+  public grantReadStreamData(principal?: iam.IPrincipal) {
```

Comment:
I guess this should be grantStreamReadData (do you think “data” is really needed?)